WorldModel/WorldModel/Core/Core.aslx

```diff
@@ -126,6 +126,16 @@
       else {
         JS.setCss ("#status", "border:none")
       }
+      // Pertex: this is new
+      if (HasString(game, "locationbarimage")){
+        url = game.locationbarimage
+        if (not Trim(url) = ""){ 
+          if (not StartsWith(game.locationbarimage,  "http")){

```

Comment:
This should check if the url starts with `http://` or `https://`

Reply:
We removed the ability to use an external URL.

If `game.locationbarimage` is an empty string (or starts with `http://` or `https://`), nothing will happen in this bit of the script now.